tests/unit/BusinessStartDate.spec.ts

```diff
@@ -87,4 +87,18 @@ describe('Business Start Date', () => {
     expect(wrapper.find('.v-chip__content').text()).toBe('Corrected')
     expect(wrapper.find('#start-undo-btn').text()).toBe('Undo')
   })
+
+  it('has correct minimum and maximum dates for a staff user ', () => {
+    store.setKeycloakRoles(['staff'])
+
+    expect(wrapper.vm.startDateMin).toBe(null) // no minimum date
+    expect(wrapper.vm.startDateMax).toBe('2021-09-29') // 90 days after founding date
```

Comment:
^^ verified

![image](https://github.com/bcgov/business-edit-ui/assets/10002889/4b9104db-6fb0-44ea-9f97-550866b72e98)

Reply:
Should shouldn't the end date be included?

Reply:
The date math we were told is "date + 90".

@Mihai-QuickSilverDev is this OK with you?